### DISCUSSION TITLE
YouTube Channel/Tutorials

### DISCUSSION THREAD

**Discussion by DavidWalshe93** [reflex-dev/reflex#1883] [Ideas]
Reflex is awesome, highly appreciate all the great work and care put into this project.

As an idea, I think having a YouTube channel with tutorials would be great. 

Docs are generally great but I feel a video course where the development team can discuss best practices and design patterns using Reflex would be great. Things like project layout, best practices on styling/theming, general overview around hosting/self-hosting. 

As an example, I have looked through the reflex-web repo and would love to understand the thought process of the project layout as well as why/how certain aspects of the site are designed and implemented. It would give better understanding on how to use the reflex framework to develop bigger projects, like the reflex website. 

Thanks again for all the great work in making this project a reality, Python for the win 🏆

**Comment by Alek99** [comment]
Thanks! Yes we definitely want to do this soon, working on a bunch of update right now but once the api get more stable we will for sure do this
